src/gsi/GSIReaction.h

```diff
@@ -125,17 +128,19 @@ class GSIReaction
      */
     void parseFormula(
         const Mutation::Thermodynamics::Thermodynamics& thermo,
-        const Mutation::Utilities::IO::XmlElement& node_reaction,
-        const SurfaceProperties& surf_props)
+        const SurfaceState& surf_state,
+        const Mutation::Utilities::IO::XmlElement& node_reaction)
     {
         std::string reactants;
         std::string products;
         splitFormulainReactantsProducts(reactants, products, node_reaction);
-        
-        parseSpecies(m_reactants, reactants, node_reaction, thermo, surf_props);
-        parseSpecies(m_products, products, node_reaction, thermo, surf_props);
+
+        parseSpecies(m_reactants, m_reactants_surf,
```

Comment:
You may consider using indentation that is invariant to code refactoring.  For example, if you one day change the name of `parseSpecies`, the parameters names will not line up any more.  A better scheme would be to put a newline before the first parameter and indent one level.

Reply:
Done